.travis.yml

```diff
@@ -10,23 +10,55 @@ script:
   - script/build
 xcode_workspace: ReactiveCocoa.xcworkspace
 matrix:
+  allow_failures:
+    - xcode_scheme: ReactiveCocoa-iOS
+      osx_image: xcode8
```

Comment:
Same for the Swift 3 branch. iOS passes tests locally, but cannot run at all on Travis. Compilation is successful though - it just fails right after the test executable is signed.